src/renderer/modules/webpack/common/index.ts

```diff
@@ -29,8 +29,8 @@ importTimeout("channels", import("./channels"), (mod) => (channels = mod.default
 export let constants: typeof import("./constants");
```

Comment:
Doesn't work properly in the docs https://docs.replugged.dev/variables/common.html

Reply:
(pretty sure this can be resolved with `export * as constants from "./constants"`)